packages/js/tests/analysis/isTaxonomyAnalysisActive.test.js

```diff
@@ -0,0 +1,19 @@
+import isTaxonomyAnalysisActive from "../../src/analysis/isTaxonomyAnalysisActive";
+
+describe( "A test to check whether the taxonomy assesseor is used for taxonomy terms", () => {
+	it( "is not used if the content type is not a taxonomy term", () => {
+		const isTerm = false;
+		const contentType = isTaxonomyAnalysisActive( isTerm );
+
+
+		expect( contentType ).toEqual( isTerm );
+	} );
```

Comment:
The `isTaxonomyActive` function does not get an argument, so I do not think this unit tests tests what we expect it to test.

Since the function gets the value of `isTerm` from the `window`, I would expect the test to look like this:
```js
        // Set the right data on the (mocked) window object.
		window.wpseoScriptData = {
			metabox: {
				isTerm: 0,
			},
		};

		expect( isTaxonomyAnalysisActive() ).toEqual( false );
```